tests/theia_automation_lsp/inc/theia/constants.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2019 Broadcom.
+# Copyright (c) 2020 Broadcom.
```

Comment:
Is possible (maybe in the future) split this huge file? Seems a kind of Pandora vase xD - btw I appreciate the order used to store the constants 👍